src/main/java/keeper/project/homepage/util/service/SurveyUtilService.java

```diff
@@ -50,7 +50,7 @@ public SurveyReplyExcuseEntity generateSurveyReplyExcuse(
         .restExcuse(because)
         .build();
 
-    surveyMemberReplyEntity.assignSurveyReplyExcuseEntity(excuse);
+    surveyMemberReplyEntity.setSurveyReplyExcuseEntity(excuse);
```

Comment:
커멘트를 달고 싶은데 윗부분이 잘려서 아래와 같이 코드를 붙여넣겠습니다!

```java
public SurveyReplyExcuseEntity generateSurveyReplyExcuse(
  SurveyMemberReplyEntity surveyMemberReplyEntity, String because) {
SurveyReplyExcuseEntity excuse = SurveyReplyExcuseEntity.builder()
    .surveyMemberReplyEntity(surveyMemberReplyEntity)
    .restExcuse(because)
    .build();

surveyMemberReplyEntity.setSurveyReplyExcuseEntity(excuse);
```
이 부분을 연관 관계 편의 메서드를 사용해 작성할 수 있지 않을까요?

Reply:
아 이거 스터디에서 이야기 나왔을때 적용해야겠다고 생각했었는데 깜빡했네요 수정하겠습니다!